brokers/unified/broker.go

```diff
@@ -48,20 +48,25 @@ type Broker struct {
 }
 
 // NewBroker creates Che broker instance
-func NewBroker(localhostSidecar bool) *Broker {
-	commonBroker := common.NewBroker()
-	ioUtils := utils.New()
-	storageObj := storage.New()
-	httpClient := &http.Client{}
-	rand := common.NewRand()
-
-	vscodeBroker := vscode.NewBrokerWithParams(commonBroker, ioUtils, storageObj, rand, httpClient, localhostSidecar)
-	return &Broker{
-		Broker:       commonBroker,
-		Storage:      storageObj,
-		utils:        ioUtils,
-		vscodeBroker: vscodeBroker,
+func NewBrokerWithParams(
```

Comment:
Please update the godoc comment (first word should match function name)

Reply:
Fixed in commit https://github.com/eclipse/che-plugin-broker/pull/62/commits/8587d25e8ab82466cb1ca0aaff5c0a083efa882f